responsiveness-homework-starter-code/index.html

```diff
@@ -32,26 +32,29 @@
 
   <div class="main-content">
     <header>
-      <div class="containerimg">
+      <div class="containerimg  ``">
         <img id="headshot" src="images/headshot.jpg">
       </div>
 
       <div class="right">
-        <h1>Stephen Reid</h1>
-        <h2>Devops Engineer</h2>
-        <div  class="containertext">
-          <p>University of Portsmouth</p>
-          <p>BSc Economics, Finance and Banking 2:1</p>
-        </div>
+
+
 
         <div class="containertext">
+
+                  <div  >
```

Comment:
this should be <div>, not <div > with a space.

Reply:
It was about div tag (there shouldn't be any spaces).